src/tape_drivers/ibm_tape.c

```diff
@@ -193,6 +193,11 @@ DRIVE_DENSITY_SUPPORT_MAP jaguar_drive_density_strict[] = {
 };
 
 DRIVE_DENSITY_SUPPORT_MAP lto_drive_density[] = {
+	/* LTOA */
+	{ DRIVE_GEN_LTOA, TC_MP_LTOPAD_CART, TC_DC_LTOPA,  MEDIUM_PERFECT_MATCH},
```

Comment:
I believe `{ DRIVE_GEN_LTOA, TC_MP_LTOPAD_CART, TC_DC_UNKNOWN, MEDIUM_PERFECT_MATCH},` is needed.